src/compiler/repositories.md

```diff
@@ -0,0 +1,25 @@
+# Repositories maintained by the Compiler Team
+
+The obvious focus of the Compiler Team is the [rustc] compiler itself, but the team also maintains a few other repositories:
+
+- An experimental [C codegen backend][c-backend] for rustc, which lowers Rust MIR to C code and compiles it with a C compiler. Currently maintained by X and Y.
+- A [Cranelift][cranelift-backend] experimental rustc backend, based on [Cranelift]. This has the potential to improve compilation times in debug mode. Currently maintained by X and Y.
+- A Rust port of the [LLVM::APFloat][rustc_apfloat] library. Maintained by X abnd Y.
+- A fork of [Enzyme], a high-performance automatic differentiator of LLVM and MLIR (more info at this [link][enzyme-mit]). The fork is currently maintained by X and Y.
+- [Thorin], a DWARF packaging utility supporting GNU extension and DWARF 5 package formats. Maintained by X and Y.
```

Comment:
Remark: I'm not sure... I seem to recall maybe @davidtwco and @nagisa has been primarily maintaining it?